app/data/store/MongoStore/MCategoryStore.scala

```diff
@@ -0,0 +1,88 @@
+package data.store.MongoStore
+
+import java.util
+
+import data.{MorphiaDAO, StoreProvider}
+import data.entity.Category
+import data.store.CategoryStore
+import javax.inject.Inject
+import org.bson.types.ObjectId
+
+/**
+  * Mongo ExpensesStore
+  */
+class MCategoryStore @Inject()(private val storeProvider: StoreProvider) extends CategoryStore{
```

Comment:
@Bukarachka there should be no duplications in store classes, let's make a generic one and reuse when possible